lib/Varien/Io/File.php

```diff
@@ -554,29 +565,41 @@ protected function _checkSrcIsFile($src)
      */
     public function filePutContent($filename, $src)
     {
-        @chdir($this->_cwd);
+        if ($this->_cwd) {
+            @chdir($this->_cwd);
+        }
         $result = @file_put_contents($filename, $src);
-        chdir($this->_iwd);
+        if ($this->_iwd) {
+            chdir($this->_iwd);
+        }
```

Comment:
Perhaps I can `@` for `chdir` like others.

Reply:
Your solution is fine. No need to change.